src/nft/types/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './checkout'
```

Comment:
nit: If everything in this folder is types, can you add a lint rule so that all of the imports in here include `import type`? I _think_ this will improve compilation speed.

Reply:
Does this include if there are enums and interfaces?

Reply:
I think so, but I'm not sure. It's ok to leave this as-is.